tests/data/region_processing/external_repo_test/nomenclature.yaml

```
@@ -11,4 +11,7 @@ definitions:
   variable:
     repository: common-definitions
 mappings:
-  repository: common-definitions
+  repository:
+    name: common-definitions
+    include:
+      - MESSAGEix-GLOBIOM 2.1-M-R12
```

I'm curious if this works if you import a mapping that is defined in a file with multiple models...
```suggestion
      - REMIND-MAgPIE 3.1-4.6
```

---

I did take that into account when writing the feature so in principle it should work. 
Good to test it explicitly though :+1: